internal/convert/converter.go

```diff
@@ -554,6 +552,55 @@ func (c Converter) convertStruct(name string, fromVal, toVal reflect.Value) erro
 	}
 }
 
+func (c Converter) convertInterface(name string, fromVal, toVal reflect.Value) error {
+	switch fromVal.Kind() {
+	case reflect.Map:
```

Comment:
is it possible just creating a Map[string]any and then return the result from convert with new created map?

Reply:
Oh... Right, it is. Haste makes waste.